dashboard/app/models/user.rb

```diff
@@ -138,6 +138,9 @@ class User < ApplicationRecord
   PROVIDER_SPONSORED = 'sponsored'.freeze # "new" user created by a teacher -- logs in w/ name + secret picture/word
   PROVIDER_MIGRATED = 'migrated'.freeze
 
+  # Countries that require a 14 character password minimum
+  STRICT_PASSWORD_COUNTRIES = %w[AU NZ RD].freeze
```

Comment:
Prefix with `PASSWORD`to group all password-related variables together:
```suggestion
  PASSWORD_STRICT_COUNTRIES = %w[AU NZ RD].freeze
```